utils/users.js

```diff
@@ -8,8 +8,13 @@ const { fetchUser } = require('../models/users')
  */
 const getUserId = async (username) => {
   try {
-    const { user: { id } } = await fetchUser({ username })
-    return id
+    const user = await fetchUser({ username })
+
+    if (user.userExists) {
+      return user.user.id
+    } else {
+      return false
+    }
```

Comment:
Could we use destructuring here?

```JS
const { userExists, user: { id } } = await fetchUser({ username })
```

Also, the return statements could be simplified with ternary operator:
```JS
return userExists ? id : false
```

What do you think which one is easy to read?

Reply:
Actually @ankushdharkar previously [commented ](https://github.com/Real-Dev-Squad/website-backend/pull/338#discussion_r619690834). So I thought we can go ahead without destructuring.  As we are just returning the id. Let me know what to finalize.

Reply:
To answer his question, yes, we're returning the const variable
He didn't say anything about the destructuring, did he? 🤔

Reply:
I answered it No and did the changes. As we are not using that `const` it anywhere in function further then why to store it in variable. 
I am ok with both the approach as it won't need much of changes. Just wanted to clarify why I did it.